packages/pangea-node-sdk/src/utils/utils.ts

```diff
@@ -135,7 +135,14 @@ export function getCustomSchemaTestToken(environment: string) {
   return process.env[name] || "";
 }
 
-export function getFileUploadParams(file: string | Buffer): FileScan.ScanFileParams {
+export async function getCRC32C(data: Buffer): Promise<string> {
+  const uint8Buffer = new Uint8Array(data);
+  const c = await crc32c(uint8Buffer);
+
+  return c;
```

Comment:
Could lose the variable:
```suggestion
  return await crc32c(uint8Buffer);
```